tfhe/src/core_crypto/fft_impl/fft64/crypto/wop_pbs/mod.rs

```diff
@@ -551,7 +550,11 @@ pub fn cmux_tree_memory_optimized<Scalar: UnsignedTorus + CastInto<usize>>(
                         ciphertext_modulus,
                     );
 
-                    if j != nb_layer - 1 {
+                    if j == nb_layer - 1 {
```

Comment:
this will happen only once and is therefore not the interesting case, we can change the comparison to j < nb_layer - 1 I think